### ISSUE TITLE
Numbers, currency, and locale hint

### ISSUE THREAD

**Issue by AutoSponge** [w3c/pronunciation#137]
Looking back at https://github.com/w3c/pronunciation/wiki/%5BDRAFT%5D-ARIA-Counter-Proposal#say-as. I was reminded of this by an issue on web-a11y-slack. The locale format for a currency used `.` instead of `,` as the separator and AT wasn't producing an expected result. This is a solid use case. Should probably add it in.
